src/app/base/components/node/StorageTables/AvailableStorageTable/AvailableStorageTable.tsx

```diff
@@ -286,8 +287,9 @@ const AvailableStorageTable = ({
   const dispatch = useDispatch();
   const [expanded, setExpanded] = useState<Expanded | null>(null);
   const [selected, setSelected] = useState<(Disk | Partition)[]>([]);
-  const [bulkAction, setBulkAction] = useState<BulkAction | null>(null);
+  const [, setBulkAction] = useState<BulkAction | null>(null);
```

Comment:
If `bulkAction` was only used for deciding whether or not actions should be disabled, and that value is now unneeded, we should also be able to remove all calls for `setBulkAction` without any side effects, and then we can remove this state entirely.